### DISCUSSION TITLE
【不具合？】購読中の質問への回答にコメントがつくと、自分の回答へのコメントでなくても「XXXがあなたの回答にコメントしました」という通知が来る

### DISCUSSION THREAD

**Discussion by Yotty0404** [increments/qiita-discussions#191] [要望・アイデア・不具合]
普段からQiitaの質問を利用させていただいているのですが、通知の内容で正しくないと思えるものがありましたので投稿させていただきました。
Qiita Supportから直接お問い合わせもしましたが、こちらでも投稿させていただきます。

表題の通りですが、購読中の質問への回答にコメントがつくと、自分の回答へのコメントでなくても「XXXが**あなたの**回答にコメントしました」という通知が届きます。
正しくは「購読中の質問にコメントが付きました」や「購読中の質問のXXXさんの回答に、XXXがコメントしました」などではないでしょうか？
本当に自分の回答にコメントがついたときと区別がつかないので、可能であれば修正していただきたいです。

具体例としては、以下のようなパターンで起こります。

■パターン1
質問に回答する（ここで質問が購読中になる）。
同じ質問に別の方が回答する。
質問者が別の方の回答にコメントをつける。
**→自分の回答にはコメントがついていなくても「XXXがあなたの回答へコメントしました」という通知が届きます。**

■パターン2
気になった質問があったので「質問を購読する」をクリック。
誰かがその質問へ回答する。
質問者がその回答にコメントをつける。
**→その質問へは回答すらつけていないのに「XXXがあなたの回答へコメントしました」という通知が届きます。**

**Comment by kyntk** [comment]
@Yotty0404 ご報告ありがとうございます。
不具合の可能性が高いので、issueを作成し、調査と対応します！ #193

**Reply by Yotty0404** [reply]
承知しました、ありがとうございます！

**Comment by mkdk09** [comment]
@Yotty0404
こちら修正が完了しました！
通知文言を`XXXが質問「○○○」にコメントしました。`に変更しました。
大変お待たせしました！🙇

**Reply by Yotty0404** [reply]
ご対応ありがとうございます！
